### PR TITLE
add two files with homework

### PR DESCRIPTION
**Pull request by Natali-Pchelkina**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (ghost) [inline about/index.html:1]: Why your file index.html in the root directory is empty?

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<html>
```
Comment (ghost) [inline about/index.html:2]: Please make your HTML prettier. Follow that rule https://github.com/hail2u/html-best-practices#other.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <head>
+        <meta charset="utf-8">
```
Comment (ghost) [inline about/index.html:4]: `meta` tag should be closed.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <title>Об авторе</title>
+    </head>
+    <body>
+		<h1><ins>Как я решила попробовать себя во фронтенд-разработке</ins></h1>
+		    <p>Моя история началась c того, что полтора года назад я устроилась на работу специалистом в 
+            <b>Банк</b>, а если точнее, то в <b>департамент информационных технологий</b>.
+		<br>
+		Первоначально я там ничего и никого не понимала и чувствовала себя совершенно <em>"глупой девочкой"</em>. 
+        Это и подтолкнуло меня попробовать себя в айти</p>
+		<hr>
```
Comment (ghost) [inline about/index.html:14]: See previous comment.

File: about/index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Об авторе</title>
+    </head>
+    <body>
+		<h1><ins>Как я решила попробовать себя во фронтенд-разработке</ins></h1>
+		    <p>Моя история началась c того, что полтора года назад я устроилась на работу специалистом в 
+            <b>Банк</b>, а если точнее, то в <b>департамент информационных технологий</b>.
+		<br>
```
Comment (ghost) [inline about/index.html:11]: See previous comment.

File: about/index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+		Первоначально я там ничего и никого не понимала и чувствовала себя совершенно <em>"глупой девочкой"</em>. 
+        Это и подтолкнуло меня попробовать себя в айти</p>
+		<hr>
+		<h2><ins>Выбор направления</ins></h2>
+		    <p>Начались поиски направления. Сначало думала стать тестировщиком ПО.
+            Но поняла, что пока мне это <del>не интересно</del>. И тут в <a href="https://www.youtube.com/">YouTube</a> 
+            я наткуналсь на видео <mark> "кто такой верстальщик"</mark></p>
+		        <p>Началось изучение...</p>
+		<hr>
```
Comment (ghost) [inline about/index.html:20]: See previous comment.

File: about/index.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+		        <li>бесплатные курсы - пустая трата времени</li>
+		        <li>платные курсы. Как итог - информации много, структуры нет, разъяснение уже для всё знающих, 
+            непонимание накапливалось как снежный ком. Интерес стал пропадать, хотелось все бросить.
+		Но зато появился аккаунт на
+         <a href="https://github.com/Natali-Pchelkina">Github</a>.</li>
+			    <li>этап приведения мыслей в порядок и спокойствия</li>
+			    <li>курсы в школе Intellect Fox. Пока очень довольна. &#128293 
+                Начинаю понимать то, чего не смогла понять за последние полтора месяца &#9757</li>
+			</ul>
+		<hr>
```
Comment (ghost) [inline about/index.html:35]: See previous comment.

File: about/index.html (hunk 1 above)
Comment (Natali-Pchelkina) [reply about/index.html:1]: Fixed error in tag <html lang="ru">

File: about/index.html (hunk 2 above)
Comment (Natali-Pchelkina) [reply about/index.html:2]: Leveled

File: about/index.html (hunk 3 above)
Comment (Natali-Pchelkina) [reply about/index.html:4]: Closed the tag

File: about/index.html (hunk 4 above)
Comment (Natali-Pchelkina) [reply about/index.html:14]: Closed the tag

File: about/index.html (hunk 5 above)
Comment (Natali-Pchelkina) [reply about/index.html:11]: Closed the tag

File: about/index.html (hunk 6 above)
Comment (Natali-Pchelkina) [reply about/index.html:20]: Closed the tag

File: about/index.html (hunk 7 above)
Comment (Natali-Pchelkina) [reply about/index.html:35]: Closed the tag

**Pull request closed without merging**